Sources/App/configure.swift

```diff
@@ -22,12 +22,16 @@ public func configure(_ app: Application) throws {
         app.logger.error("Incomplete DB configuration:\n\(vars)")
         throw Abort(.internalServerError)
     }
-    
+
+    let tlsConfig: TLSConfiguration? = Environment.get("DATABASE_USE_TLS")
+        .flatMap(\.asBool)
+        .flatMap { $0 ? .clientDefault : nil }
     app.databases.use(.postgres(hostname: host,
                                 port: port,
                                 username: username,
                                 password: password,
-                                database: database), as: .psql)
+                                database: database,
+                                tlsConfiguration: tlsConfig), as: .psql)
```

Comment:
`nil` is the default, so this doesn't change anything for `production` and since the CI tests pass (which don't have `DATABASE_USE_TLS` set either and use a non-TLS db this shows that it doesn't affect production deployments.